src/rolch/link/generic.py

```diff
@@ -0,0 +1,47 @@
+from typing import Tuple
+
+import numpy as np
+
+from rolch.base import LinkFunction
+
+
+class GenericInverseLink(LinkFunction):
+    """
+    This link function maps an arbitrary link function to its inverse by swapping the links and derivatives.
+    You need to provide the link function and the support.
+    This can be used to quickly create inverse links or as base class if you want to implement a custom link function.
+    The default will not provide the second derivative, which you need to implement in the subclass.
+    """
+
+    def __init__(
+        self,
+        link_function: LinkFunction,
+        link_support: Tuple,
+    ) -> None:
+        """Initializes the GenericInverseLink class.
+
+        Args:
+            link_function (LinkFunction): The Link function to invert
+            link_support (Tuple): The support of the link function
```

Comment:
This is misleading, we want to pass the support of the *inverted* link function here, right?
```suggestion
            link_support (Tuple): The support of the inverted link function
```